project/profiles/capacity4more/modules/c4m/content/c4m_content_wiki_page/c4m_content_wiki_page.features.field_base.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @file
+ * c4m_content_wiki_page.features.field_base.inc
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function c4m_content_wiki_page_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'field_c4m_editing_open'.
+  $field_bases['field_c4m_editing_open'] = array(
```

Comment:
The custom table is the only option to extend this later on.
As we know this day will come, this should be the way to go imo.